eclipse-collections-code-generator/src/main/resources/test/factory/primitiveObjectMapsTest.stg

```diff
@@ -0,0 +1,69 @@
+import "copyright.stg"
+import "primitiveLiteral.stg"
+
+isTest() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/impl/map/primitive"
+
+fileName(primitive) ::= "<primitive.name>ObjectMapsTest"
+
+skipBoolean() ::= "true"
+
+class(primitive) ::= <<
+<body(primitive.type, primitive.name, primitive.wrapperName, primitive.charPrimitive)>
+>>
+
+body(type, name, wrapperName, charPrimitive) ::= <<
+<copyright()>
+
+package org.eclipse.collections.impl.map.primitive;
+
+import org.eclipse.collections.impl.factory.primitive.<name>ObjectMaps;
+import org.eclipse.collections.impl.map.mutable.primitive.<name>ObjectHashMap;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**mvn
```

Comment:
Looks like there is a stray `mvn` here in this block comment.

Reply:
ops - my bad, fixed

Reply:
I don't see it reflected in the PR or your fork yet, so it might not have been pushed yet. I'll resolve this as soon as I see it though! 👍

Reply:
@rinatigati hey apologies - just pushed the changes the, the ack for the issue was only reflected on my local branch as I was still not ready to repush